### PR TITLE
[CentOS Stream] support centOS Stream in ansible.

### PR DESCRIPTION
**Pull request by ZouYuhua**
Task:
support centOS Stream in ansible.

Result:
1) passed to test CentOS-Stream-9-x86_64-latest-dvd1.iso  with RHEL/8/server_with_GUI/ks.cfg and EFI firmware
image

2) passed to test CentOS-Stream-9-x86_64-latest-dvd1.iso  with RHEL/8/server_without_GUI/ks.cfg and EFI firmware
image

3) passed to test CentOS-Stream-9-x86_64-latest-dvd1.iso with RHEL/8/server_with_GUI/ks.cfg and bios firmware
image

4) passed to test CentOS-Stream-8-x86_64-latest-dvd1.iso  with RHEL/8/server_with_GUI/ks.cfg and bios firmware
image

5) passed to test CentOS-Stream-8-x86_64-latest-dvd1.iso  with RHEL/8/server_without_GUI/ks.cfg and EFI firmware
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ This project supports below scenarios for end-to-end guest operating system vali
 |:-------------------------------------------------------| :------------------------------: | :----------------------: | :--------------------------------: |
 | Red Hat Enterprise Linux 7.x, 8.x, 9.x                 | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
 | CentOS 7.x, 8.x                                        | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| CentOS Stream 9.x                                      | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
```
Comment (keirazhang) [inline README.md:73]: There is no CentOS Stream 9.x, only 9. What about CentOS Stream 8?

File: autoinstall/README.md [hunk 2]
```diff
@@ -27,6 +27,7 @@
 26. For FusionOS 22 unattend auto-install, please use files under Fusion.
 27. For FusionOS 23 unattend auto-install, please use file Fusion/server_without_GUI/ks.cfg.
 28. For Kylin Linux Advanced Server V10 unattend auto-install, please use file Kylin/Server/10/ks.cfg.
+29. For CentOS Stream 9.x or later unattend auto-install, please use files under RHEL/8.
```
Comment (keirazhang) [inline autoinstall/README.md:30]: Please replace `9.x` with `9`

**Comment by ZouYuhua** [comment]
@keirazhang  tests are passed for CentOS Stream 8 and 9.   Thanks

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**